src/alire/alire-origins-deployers.adb

```diff
@@ -117,7 +117,12 @@ package body Alire.Origins.Deployers is
    function Deploy (From : Origin; Folder : String := "") return Outcome is
    begin
       if From.Is_Native then
-         return Install_Native (From);
+         if Platform.Distribution_Is_Known then
+            return Install_Native (From);
+         else
+            return Outcome_Failure
+              ("Native packages are unavailable in unknown distributions");
```

Comment:
What about this error message: `Cannot provide native package for unknown distribution`?
I think it explain better that it is because the distribution is unknown that we cannot provide the package.
Or: `Unknown distribution: Cannot provide native package`

Also print some kind of name for the native dependency.

Reply:
Yup, better.

Reply:
Fixed in 3f38dfc